presentation/src/main/kotlin/com/jshvarts/conductormvp/notes/NotesView.kt

```diff
@@ -38,9 +40,11 @@ class NotesView : BaseView(), NotesContract.View {
 
     override fun onAttach(view: View) {
         super.onAttach(view)
-        initRecyclerView(view.context)
-        presenter.start(this)
-        presenter.loadNotes()
+        recyclerView.initRecyclerView(LinearLayoutManager(view.context), recyclerViewAdapter)
```

Comment:
You can use `app:layoutManager="path.to.layout.mgr.LayoutManager"` in the recycler view xml instead of needing to manually specifiy it here.

Reply:
ran into issues using this